json/products/mozilla-concrete-mortar.json

```diff
@@ -1,8 +1,11 @@
 {
   "description": {
-    "en": "Some mortar"
+    "en": "Mortar"
   }, 
   "img": "https://raw.githubusercontent.com/mozilla/payments-config/master/payments_config/assets/mortar.png", 
+  "price": {
```

Comment:
having price in addition to ``amount`` is confusing. Is there an issue to get rid of ``amount`` later?